### PR TITLE
Localize command.x cont 

### PR DESCRIPTION
**Pull request by JavierMatosD**
This PR localizes all the messages in the following files:
- commands.dependinfo.cpp
- commands.create.cpp
- commands.hash.cpp

### CODE REVIEW

File: src/vcpkg/commands.dependinfo.cpp [hunk 1]
```diff
@@ -320,10 +315,11 @@ namespace vcpkg::Commands::DependInfo
             provider, var_provider, specs, status_db, {host_triplet, UnsupportedPortAction::Warn});
         for (const auto& warning : action_plan.warnings)
         {
-            print2(Color::warning, warning, '\n');
+            msg::write_unlocalized_text_to_stdout(Color::warning, warning + '\n');
         }
-        Checks::check_exit(
-            VCPKG_LINE_INFO, action_plan.remove_actions.empty(), "Only install actions should exist in the plan");
+
+        Checks::msg_check_exit(VCPKG_LINE_INFO, action_plan.remove_actions.empty(), msgInvalidActionsInstall);
```
Comment (BillyONeal) [inline src/vcpkg/commands.dependinfo.cpp:321]:
```suggestion
        if (!action_plan.remove_actions.empty()) {
            Debug::println("Only install actions should exist in the plan");
            Checks::unreachable(VCPKG_LINE_INFO);
        }
```

and remove InvalidActionsInstall please.

File: src/vcpkg/commands.dependinfo.cpp [hunk 2]
```diff
@@ -341,7 +337,7 @@ namespace vcpkg::Commands::DependInfo
                 });
 
             const std::string graph_as_string = create_graph_as_string(options.switches, depend_info);
-            print2(graph_as_string, '\n');
+            msg::write_unlocalized_text_to_stdout(Color::none, graph_as_string + '\n');
```
Comment (BillyONeal) [inline src/vcpkg/commands.dependinfo.cpp:340]:
```
            std::string graph_as_string = create_graph_as_string(options.switches, depend_info);
            graph_as_string.push_back('\n');
            msg::write_unlocalized_text_to_stdout(Color::none, graph_as_string);
```

File: include/vcpkg/base/messages.h [hunk 3]
```diff
@@ -892,6 +894,15 @@ namespace vcpkg
                     "",
                     "invalid argument: binary config '{binary_source}' requires a SAS token without a "
                     "preceeding '?' as the second argument");
+    DECLARE_MESSAGE(InvalidCommandArgMaxDepth, (), "", "Value of --max-depth must be an integer.");
```
Comment (BillyONeal) [inline include/vcpkg/base/messages.h:897]: Does it make sense to make this one reusable? e.g.

DECLARE_MESSAGE(OptionMustBeInteger, (msg::option), "", "Value of --{option} must be an integer.");

(Maybe consider alongside your vcpkgcmdarguments.cpp PR)

File: src/vcpkg/commands.dependinfo.cpp [hunk 4]
```diff
@@ -237,8 +233,7 @@ namespace vcpkg::Commands::DependInfo
                                           std::map<std::string, PackageDependInfo>& dependencies_map)
         {
             auto iter = dependencies_map.find(package);
-            Checks::check_exit(
-                VCPKG_LINE_INFO, iter != dependencies_map.end(), "Package not found in dependency graph");
+            Checks::msg_check_exit(VCPKG_LINE_INFO, iter != dependencies_map.end(), msgPackageNotFoundDependencyGraph);
```
Comment (BillyONeal) [inline src/vcpkg/commands.dependinfo.cpp:236]:
```suggestion
            if (iter == dependencies_map.end()) {
                Debug::println("Not found in dependency graph: ", package);
                Checks::unreachable(VCPKG_LINE_INFO);
             }
```

?

File: src/vcpkg/commands.dependinfo.cpp [hunk 5]
```diff
@@ -397,16 +401,14 @@ namespace vcpkg::Commands::DependInfo
 
                     if (show_depth)
                     {
-                        print2(Color::error, "(", info.depth, ") ");
+                        msg::write_unlocalized_text_to_stdout(Color::error, fmt::format("({} )", info.depth));
```
Comment (ras0219-msft) [inline src/vcpkg/commands.dependinfo.cpp:404]:
```suggestion
                        msg::write_unlocalized_text_to_stdout(Color::error, fmt::format("({}) ", info.depth));
```

**Review by ras0219-msft — APPROVED** [review]

**Pull request merged**